task_2_hw_1.py

```diff
@@ -0,0 +1,29 @@
+# 2.Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей. 
+# Дано a, b, c - стороны предполагаемого треугольника. 
+# Требуется сравнить длину каждого отрезка-стороны с суммой двух других. 
+# Если хотя бы в одном случае отрезок окажется больше суммы двух других, 
+# то треугольника с такими сторонами не существует. 
+# Отдельно сообщить является ли треугольник разносторонним, равнобедренным или равносторонним.
+
+TEXT = f"Введите длинну стороны треугольника "
+NOT_EXIST = "Tреугольника с такими сторонами не существует!"
+a = int(input(TEXT + "№1: "))
+b = int(input(TEXT + "№2: "))
+c = int(input(TEXT + "№3: "))
+
+check1 = a + b
+check2 = a + c
+check3 = b + c
+
+if (a > 0 and b > 0 and c > 0):
+    if (check1 < c or check2 < b or check3 < a):
```

Comment:
Эти 2 условия можно объединить.
```suggestion

if check1 <= c or check2 <= b or check3 <= a:
```